src/TelemetryServer.ts

```diff
@@ -13,22 +13,22 @@ interface Logger {
  * The object should have a key 'info' which is a function that takes a single parameter containing the log message.
  */
 export default class TelemetryServer {
-  private _live: boolean
-  private _ready: boolean
-  private readonly _logger: Logger
-  private readonly _app: Application
-  private _server: Server | null
+  #live: boolean
```

Comment:
Very nice, this is some new private syntax that didn't exist back when I last used TypeScript!

Reply:
Looks like it was added in ES2022, so quite recent :D